pyamg/multilevel.py

```diff
@@ -596,6 +767,19 @@ def __solve(self, lvl, x, b, cycle):
 
         self.levels[lvl].postsmoother(A, x, b)
 
+    def visualize_coarse_grids(self, directory):
```

Comment:
I didn't notice this nor remember where it came from, can probably be dropped from PR unless you guys feel otherwise

Reply:
Yep, agreed.  I don't see a reason to have vis in there.